### ISSUE TITLE
Lareevaar assist + Vishraams

### ISSUE THREAD

**Issue by tarunsingh5** [KhalisFoundation/sttm-web#1109]
turning on vishraams turns off lareevar assist - we should support both being on

**Comment by saintsoldierx** [comment]
![image](https://user-images.githubusercontent.com/27791330/95647552-24d81080-0a85-11eb-82e4-87a1a8e460bc.png)

**Comment by plug-n-play** [comment]
Still can't  turn both larivaar assist and vishraam options together. When trying to turn on one, other gets turned off.

**Comment by saintsoldierx** [comment]
@gurjit03

**Comment by saintsoldierx** [comment]
I don't think we actually have done this yet? I may have mistakenly marked it as fixed. Gurjit, please confirm

**Comment by gurjit03** [comment]
@saintsoldierx @plug-n-play haven't worked on this yet.

**Comment by gurjit03** [comment]
@saintsoldierx needs some inputs on UI here.

**Comment by saintsoldierx** [comment]
I gave UI input above @gurjit03 - rather than a circle, let's make it a solid square, orange or red.

**Comment by gurjit03** [comment]
@saintsoldierx cool thanks, I will get on this !

**Comment by Gauravjeetsingh** [comment]
I had a slightly different idea for this, shared on slack, posting here for reference:

How about using the text color for vishraams, and use the background color for assist. Something like this:
![image](https://github.com/KhalisFoundation/sttm-web/assets/1990932/fc44e51d-8c61-4924-b5de-e62b749dad4c)

Another idea, I had was using text overline for larivaar assist, but I personally liked background color more.
![image](https://github.com/KhalisFoundation/sttm-web/assets/1990932/c68d821a-c747-451d-93e8-930ea7a12da7)

**Comment by Gauravjeetsingh** [comment]
Also, whatever we finalize here, we should do the same in sttm-desktop to maintain the consistency.
